src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -55,6 +55,10 @@
     <Compile Remove="coreclr\Math\Functions\Single\**" />
   </ItemGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netcoreapp2.0' Or '$(TargetFramework)' == 'netcoreapp2.1' Or '$(TargetFramework)' == 'netcoreapp2.2' Or '$(TargetFramework)' == 'netcoreapp3.0'">
```

Comment:
Wouldn't it be easier to just exclude .NET Framework?
```xml
<ItemGroup Condition=" '$(TargetFrameworkIdentifier)' != '.NETFramework'>
```

Reply:
Right.